easybuild/easyconfigs/o/OVITO/OVITO-2.9.0-GCCcore-6.4.0-x86_64.eb

```diff
@@ -0,0 +1,29 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'PackedBinary'
+
+name = 'OVITO'
+version = '2.9.0'
+versionsuffix = '-x86_64'
+
+homepage = 'https://www.ovito.org/'
+
+description = """
+ OVITO is a scientific visualization and analysis software
+ for atomistic simulation data
+"""
+
+source_urls = [('https://www.ovito.org/download/%(version)s')]
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['2eb484483dfe97003d0c1885899abdeb3dea00499a7a186b2f8811aa91d62052']
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+sanity_check_paths = {
+    'files': ['bin/ovito', 'include/python3.5m/Python.h',
```

Comment:
Probably would..... especially if not 3.5.... 

I have another build from source in development but needs lots of work.  Just thought I'd toss this out for now (seems to have made our user happy.... at least they haven't complained).